cmd/interchaincmd/messengercmd/deploy.go

```diff
@@ -152,20 +152,20 @@ func CallDeploy(_ []string, flags DeployFlags, network models.Network) error {
 			return err
 		}
 	}
-	var teleporterVersion string
+	var icmVersion string
 	switch {
 	case flags.MessengerContractAddressPath != "" || flags.MessengerDeployerAddressPath != "" || flags.MessengerDeployerTxPath != "" || flags.RegistryBydecodePath != "":
 		if flags.MessengerContractAddressPath == "" || flags.MessengerDeployerAddressPath == "" || flags.MessengerDeployerTxPath == "" || flags.RegistryBydecodePath == "" {
-			return fmt.Errorf("if setting any teleporter asset path, you must set all teleporter asset paths")
+			return fmt.Errorf("if setting any ICM asset path, you must set all ICM asset paths")
 		}
 	case flags.Version != "" && flags.Version != "latest":
-		teleporterVersion = flags.Version
+		icmVersion = flags.Version
 	default:
-		teleporterInfo, err := teleporter.GetInfo(app)
+		icmInfo, err := teleporter.GetInfo(app)
```

Comment:
"github.com/ava-labs/avalanche-cli/pkg/teleporter" - should be renamed

Reply:
done